poglink/cogs/rates.py

```diff
@@ -142,6 +142,7 @@ async def on_ready(self):
                 last_rates = RatesStatus.from_dict(last_rates_dict)
 
                 # compare rates to last rates
+                # TODO: I think this is backwards; it should be last_rates.get_diff(rates)
```

Comment:
Thoughts on this @FM-17 ?

Reply:
Agreed
Within `get_diff`, `old = self.to_dict`, where `self` is a `RatesStatus` object, which corresponds to `<this part>.get_diff`, so `this_part` should be `last_rates`

Reply:
Okay, will update to fix

Reply:
Hmm so is it bolding the correct values as-is?